cfgov/unprocessed/css/enhancements/tables.less

```diff
@@ -120,44 +120,36 @@
         }
     }
 
-    .respond-to-max(960px, {
-
-        font-size: 14px;
-
-        thead {
-            display: none;
-        }
+}
 
-        tbody {
-            border-top: none;
+.respond-to-max( @bp-xs-max, {
+    .o-table__stack-on-small {
+        tr,
+        td {
+            display: block;
         }
 
-        tr {
-            border-bottom: none;
+        & > thead {
+          display: none
         }
 
-        td:first-child {
-            .h4();
-            margin-bottom: 10px;
-            background-color: @td-bg-mobile;
+        td[data-label]:before {
+            .webfont-demi();
+            display: block;
+            margin-top: 0;
```

Comment:
Is this reset necessary? I can't find where it's set in the code but I'm not sure if it's in our reset or a browser default.